django/http/response.py

```diff
@@ -205,7 +205,8 @@ def set_signed_cookie(self, key, value, salt='', **kwargs):
         return self.set_cookie(key, value, **kwargs)
 
     def delete_cookie(self, key, path='/', domain=None):
-        self.set_cookie(key, max_age=0, path=path, domain=domain,
+        secure = key.startswith(('__Secure-', '__Host-')) or None
```

Comment:
Why the `or None`? The default value of the `secure` argument is `False`, not `None`. `.statswith` will correctly return `False`. Or is the `None` required for a different reason that isn't immediately obvious?